arrow-core/src/main/kotlin/arrow/core/extensions/validated.kt

```diff
@@ -40,6 +41,12 @@ interface ValidatedFunctor<E> : Functor<ValidatedPartialOf<E>> {
   override fun <A, B> Kind<ValidatedPartialOf<E>, A>.map(f: (A) -> B): Validated<E, B> = fix().map(f)
 }
 
+@extension
+interface ValidatedBifunctor : Bifunctor<ForValidated> {
```

Comment:
Nice addition, thank you! Would you mind also adding the tests for BifunctorLaws in `ValidatedTest`?

Reply:
Sure

Reply:
@aballano Added the test